templates/internet-of-things/nvidia/contact-us.html

```diff
@@ -1,150 +1,261 @@
-  </div>
-</section>
+  </section>
 
-<section class="p-strip">
-  <div class="row">
-    <div class="col-8">
-      <form action="/marketo/submit" method="post" id="mktoForm_4953">
-        <fieldset class="u-no-margin--bottom">
-          <div class="js-formfield">
-            <div role="group" aria-labelledby="which-nvidia-orin-platforms">
-              <h2 class="p-heading--5" id="which-nvidia-orin-platforms">Which NVIDIA platforms are you interested in?</h2>
-              <label class="p-checkbox">
-                <input class="p-checkbox__input" type="checkbox" aria-labelledby="drive-orin" name="which-nvidia-orin-platforms" value="DRIVE Orin">
-                <span class="p-checkbox__label" id="drive-orin">Xavier</span>
-              </label>
-              <label class="p-checkbox">
-                <input class="p-checkbox__input" type="checkbox" aria-labelledby="drive-orin" name="which-nvidia-orin-platforms" value="DRIVE Orin">
-                <span class="p-checkbox__label" id="drive-orin">DRIVE Orin</span>
-              </label>
-              <label class="p-checkbox">
-                <input class="p-checkbox__input" type="checkbox" aria-labelledby="igx-orin" name="which-nvidia-orin-platforms" value="IGX Orin">
-                <span class="p-checkbox__label" id="igx-orin">IGX Orin</span>
-              </label>
-              <label class="p-checkbox">
-                <input class="p-checkbox__input" type="checkbox" aria-labelledby="jetson-agx-orin" name="which-nvidia-orin-platforms" value="Jetson AGX Orin">
-                <span class="p-checkbox__label" id="jetson-agx-orin">Jetson AGX Orin</span>
-              </label>
-              <label class="p-checkbox">
-                <input class="p-checkbox__input" type="checkbox" aria-labelledby="jetson-orin-nx" name="which-nvidia-orin-platforms" value="Jetson Orin NX">
-                <span class="p-checkbox__label" id="jetson-orin-nx">Jetson Orin NX</span>
-              </label>
-              <label class="p-checkbox">
-                <input class="p-checkbox__input" type="checkbox" aria-labelledby="jetson-orin-nano" name="which-nvidia-orin-platforms" value="Jetson Orin Nano">
-                <span class="p-checkbox__label" id="jetson-orin-nano">Jetson Orin Nano</span>
+  <section class="p-strip">
+    <div class="row">
+      <div class="col-8">
+        <form action="/marketo/submit" method="post" id="mktoForm_4953">
+          <fieldset class="u-no-margin--bottom">
+            <div class="js-formfield">
+              <div role="group" aria-labelledby="which-nvidia-orin-platforms">
+                <h2 class="p-heading--5 js-formfield-title" id="which-nvidia-orin-platforms">Which NVIDIA platforms are you interested in?</h2>
+                <label class="p-checkbox">
+                  <input class="p-checkbox__input"
+                         type="checkbox"
+                         aria-labelledby="drive-orin"
+                         value="DRIVE Orin" />
```

Comment:
```suggestion
                         value="Xavier" />
```
Shouldn't the value here be "Xavier"?

Reply:
Good catch, thanks!